monolith/views/stories.py

```diff
@@ -50,3 +56,22 @@ def _like(authorid, storyid):
     else:
         message = 'You\'ve already liked this story!'
     return _stories(message)
+
+# TODO to complete
+@stories.route('/stories/new_story', methods=['GET'])
+@login_required
+def new_stories():
+    dice_set = retrieve_dice_set()
+    themes = [dice_set.theme]
+
+    return render_template("new_story.html", themes=themes)
+
+# TODO to complete
+@stories.route('/write_story', methods=['POST'])
```

Comment:
Following the comment on /stories/new_story, this should have a GET method as well which retrieves an unpublished story, right?